update home page

ohan8 opened the pull request:
Updated homepage, removed default Next.js UI and added rotating gear icon and work in progress text. Updated Home.module.css to include only the relevant styling that is used for the current UI.

EDIT: Update screen recording

https://user-images.githubusercontent.com/28934218/200672076-143c86a7-1e2d-4c73-a778-f651f2c4e569.mov

ohan8 commented:
Duly noted and will do. Appreciate the feedback.

ohan8 commented:
Just updated commit with slower spinner speed and added animation for ellipses as well as the screen recording in the original comment to reflect these changes. Please let me know what you think.

yasaad commented on pages/index.tsx:
```diff
@@ -12,41 +14,10 @@ const Home: NextPage = () => (
+    <div className={styles.rotating}>
+      <Image src={Gear} height={300} width={300} />
+    </div>
+    <h1 className={styles.loading}>Work in progress</h1>
```
Can you please left align the text so that it does not shift when the ellipses pop in as shown in this video

https://user-images.githubusercontent.com/21961090/200678028-d3ec1cd7-2f8e-4a29-a3a0-39cc642069ed.mov

yasaad replied:
Can you please ad a smaller tag beneath the _Work in progress_ that reads: _Please come back at a later date_

ohan8 commented:
As requested, modified the text to stay in its position as the ellipses animate. Also added additional text beneath and included a media query to ensure the text looks fine on mobile as well.

https://user-images.githubusercontent.com/28934218/200681800-82446cd9-f296-4f33-bcc1-91f2163f9669.mov

![IMG_E0867C8E5F50-1](https://user-images.githubusercontent.com/28934218/200681822-14f1cadc-32bc-40ac-8bce-3456e557e40b.jpeg)

yasaad reviewed (approved):
LGTM
